python_ta/cfg/visitor.py

```diff
@@ -8,20 +8,30 @@
 class CFGVisitor:
     """An astroid visitor that creates a control flow graph for a given Python module.
 
+    Initialize with an options dict to configure the visitor.
+
+    Supported Options:
+      - "separate-condition-blocks": bool
```

Comment:
This is good, but underneath (and indented) please write a short English description of the purpose of this option.